src/main/kotlin/no/nav/familie/ba/sak/internal/GithubController.kt

```diff
@@ -0,0 +1,33 @@
+package no.nav.familie.ba.sak.internal
+
+import no.nav.familie.ba.sak.config.GithubConfig
+import no.nav.familie.kontrakter.felles.Ressurs
+import no.nav.security.token.support.core.api.ProtectedWithClaims
+import org.springframework.http.MediaType
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping(path = ["/api"], produces = [MediaType.APPLICATION_JSON_VALUE])
+@ProtectedWithClaims(issuer = "azuread")
+class GithubController(
+    private val githubConfig: GithubConfig,
+) {
+    @GetMapping("/hent-branch")
+    fun hentBranch(): ResponseEntity<Ressurs<GithubBranchDto>> {
+        val branch: String? = githubConfig.githubBranch
```

Comment:
Slik `GithubConfig` er satt opp er ikke `branch` nullable, så kan endre her til: 
```suggestion
        val branch: String = githubConfig.githubBranch
```
Eventuelt:
```suggestion
        val branch = githubConfig.githubBranch
```